src/main/java/org/itsallcode/io/CapturingOutputStream.java

```diff
@@ -88,6 +89,13 @@ public void capture()
         this.internalStream = new ByteArrayOutputStream();
     }
 
+    @Override
+    public void captureMuted()
```

Comment:
This solution is refreshingly primitive. I love it.

And here I was thinking about complicated configuration methods. :-)